Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -200,47 +213,46 @@ CGPathRef CGPathCreateCopy(CGPathRef path) {
 }
 
 /**
- @Status Interoperable
+@Status Interoperable
+@Notes
 */
 CGMutablePathRef CGPathCreateMutableCopy(CGPathRef path) {
     if (path == NULL) {
         return NULL;
     }
 
-    auto ret = __CGPath::alloc(nil);
-    ret->_max = path->_max;
-    ret->_count = path->_count;
-    ret->_elements = (CGPathElementInternal*)IwRealloc(ret->_elements, ret->_max * sizeof(CGPathElementInternal));
-    memcpy(ret->_elements, path->_elements, path->_count * sizeof(CGPathElementInternal));
-    // All of the new elements need to call init
-    // because they were created with alloc + memcpy
-    for (unsigned i = 0; i < path->_count; i++) {
-        ret->_elements[i].init();
+    CGMutablePathRef mutableRet = CGPathCreateMutable();
+
+    path->__closePath();
```

Comment:
>__closePath [](start = 10, length = 11)

Technically, we shouldn't be mutating the path here (which we are by calling __closepath().